import System.Exit (exitFailure)

import Test


main :: IO Bool
main = runTests
